### ISSUE TITLE
Darkmode Issues

### ISSUE THREAD

**Issue by bw-Deejee** [h2oai/h2ogpt#1343]
I have several issues with darkmode.

When I've set my windows sytem color mode to "dark", h2oGPT seems to adapt this.
Even though h2oGPT variable `dark: bool = False` is set, it starts up in dark mode.
If I use `--dark=True` or set  `dark: bool = True` is uses light-mode, which is super counter intuitive.

Setting my windows system color mode to "light" reverses that effect, at least making the naming now intuitive. So, it just seems as h2oGPT is expecting light mode as system default, and the variable `dark ` is simply a switch to use the non-default.

This issue seems to prevent me from forcing light mode for all users as some might use light and some use dark mode as system default.

Furthermore: when working with `--auth` and having the login mask on first access. The Login page does not obey at all to the `dark` variable and instead always uses winows system color mode. This is very annoying and I can't seem to find a way to force a light mode in the login page.

**Comment by pseudotensor** [comment]
Yes, I've noticed similar things.  I'm not js expert, I'll ask what others think.

@jefffohl Do you have any ideas?  I have same issue in sense that my desktop sticks to dark first, while other machines stick to light first.  While I can switch dark mode, I don't understand how the defaults are set.

**Comment by pseudotensor** [comment]
Related: https://github.com/h2oai/h2ogpt/issues/1303
